docs/develop/security.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: "Recipes for Confidentiality: Security considerations when writing confidential contracts"
+description: 'Recipes for Confidentiality: Security considerations when writing confidential contracts'
```

Comment:
Quotes can be removed here.

```suggestion
description: Recipes for Confidentiality: Security considerations when writing confidential contracts
```

Reply:
the Netlify deploy failed after removing quotes due to Docusaurus: https://app.netlify.com/projects/oasisprotocol-sapphire-paratime/deploys/68bebc2ea855000008cdc6c5, added it back